samples/subsys/shell/shell_module/boards/intel_socfpga_agilex5_socdk.conf

```diff
@@ -27,3 +31,37 @@ CONFIG_ARM_SIP_SVC_SUBSYS_SINGLY_OPEN=y
 
 #SiP SVC Service Shell
 CONFIG_ARM_SIP_SVC_SUBSYS_SHELL=y
+
+# Network driver config
+CONFIG_MDIO=y
+CONFIG_NET_L2_ETHERNET=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+# networking
+CONFIG_NETWORKING=y
+CONFIG_NET_TCP=y
+CONFIG_NET_UDP=y
+CONFIG_NET_MAX_CONTEXTS=10
+CONFIG_NET_BUF_RX_COUNT=1024
+CONFIG_NET_BUF_TX_COUNT=512
+CONFIG_NET_STATISTICS=y
+CONFIG_NET_STATISTICS_USER_API=y
+CONFIG_NET_SHELL=y
+#IPV4 options
+CONFIG_NET_IPV4=y
+CONFIG_NET_IF_MAX_IPV4_COUNT=3
+CONFIG_NET_CONFIG_SETTINGS=y
+CONFIG_NET_CONFIG_MY_IPV4_ADDR="10.10.0.101"
+CONFIG_NET_CONFIG_MY_IPV4_NETMASK="255.255.255.0"
+CONFIG_NET_CONFIG_MY_IPV4_GW="8.8.8.8"
+CONFIG_NET_CONFIG_PEER_IPV4_ADDR="10.10.0.102"
```

Comment:
Please use the default IP addresses here, and not the ones from you own setup.

Why do we set network options in shell sample? Shouldn't at least some of these come semi automatically from the board default config. This looks somewhat strange how this is done.

Reply:
removed these configurations

Reply:
Removed and replaced with default IP Addresses